doc/ref/preface.xml

```diff
@@ -15,9 +15,11 @@
 <Index Key="About GAP manual">About &GAP; manual</Index>
 
 Welcome to &GAP;. 
-This is one of two manuals documenting the core part of &GAP;, 
+This is one of the manuals documenting the core part of &GAP;,
 the others being the <E>&GAP; Tutorial</E>
-<Alt Only="HTML">(see <Ref BookName="tut" Label="Preface"/>)</Alt>.
+<Alt Only="HTML">(see <Ref BookName="tut" Label="Preface"/>)</Alt>
+and  <E>HPC-GAP Reference Manual</E>
+<Alt Only="HTML">(see <Ref BookName="hpc"/>)</Alt>.
```

Comment:
This fails in Travis, because of missing `Label` element. Add `"Label="Tasks"` to point it to the 1st chapter.
```
--------------------
Building GAP manuals
--------------------
----------------------------
Building GAP manual 'ref' at /home/travis/build/gap-system/gap/doc/ref
Run 1 of 2
----------------------------
#I Composing XML document . . .
#I Parsing XML document . . .
#I Checking XML structure . . .
XML Parse Error: Line 93 Character 23
Original file: /home/travis/build/gap-system/gap/doc/ref/preface.xml, line num\
ber 22.
-----------
<Alt Only="HTML">(see <Ref BookName="hpc"/>)</Alt>.
                      ^
-----------
Ref with strange attribute set: 
!!! Type 'Show();' to watch the input string in pager - starting with
    line containing error !!!
Or 'ShowOrigin();' to look it up in its source file.
Error,  at /home/travis/build/gap-system/gap/pkg/GAPDoc-1.6.3/lib/XMLParser.gi:146 called from
Makefile.rules:796: recipe for target 'doc' failed
make: *** [doc] Error 1
```